README.md

```diff
@@ -0,0 +1,20 @@
+# Remix Compute for Netlify
```

Comment:
Mind having a peek at this @stephmarie17? It's to the point, but just want to make sure about the wording.

Reply:
This looks good to me! Clear and to the point.